dapp-developer-guide/tutorials/use-ethstorage-sdk-to-upload-and-download-files.md

```diff
@@ -14,9 +14,14 @@ You can easily switch to other chains by specify a different RPC endpoint, such
 
 You can install `ethstorage-sdk` by the following command:
 
-```sh
-npm i ethstorage-sdk
-```
+- For the **Sepolia network**, install version 2:
+    ```sh
+    npm i ethstorage-sdk@2
+    ```
+- For other networks, install the latest version normally:
```

Comment:
may need to specify the SWC Beta network

Reply:
Also we may want put the SWC network at first

Reply:
Already added